scripts/Muon/GUI/ElementalAnalysis/LoadWidget/load_utils.py

```diff
@@ -37,6 +37,13 @@ def load_run(self):
             filename: get_filename(filename, self.run)
             for filename in to_load if get_filename(filename, self.run) is not None
         }
+        countWorkspaces = {}
+        unique_workspaces = {}
+        for path, workspace in workspaces.iteritems():
+            countWorkspaces[workspace] = countWorkspaces.get(workspace, 0) + 1
+            if countWorkspaces[workspace] == 1:
+                unique_workspaces[path] = workspace
+        workspaces = unique_workspaces
```

Comment:
could you not do something like this: https://www.w3schools.com/python/python_howto_remove_duplicates.asp

Reply:
I've cleaned up the loop, as the countWorkspaces was redundant.